### ISSUE TITLE
fenced ```code blocks``` handled as inline `code`

### ISSUE THREAD

**Issue by AdUser** [bobtfish/text-multimarkdown#24]
Difference:

`this is inline` code in line renders to `<code class="something">this is inline</code>`

```
this is fenced code block
surrounded by three «`»
should be rendered as <pre><code class="something">blah</code></pre>
```

See [Fenced Code Blocks](https://rawgit.com/fletcher/human-markdown-reference/master/index.html)

**Comment by jahagirdar** [comment]
as per the cheatsheet at https://rawgit.com/fletcher/human-markdown-reference/master/index.html 

Code like

``````
```perl
print "hello world";
exit(0);
```
``````

should render with appropriate syntax highlighting if highlight.js is linked to.
The expected output should be

``` perl
print "hello world";
exit(0);
```

Going through the code of Text::Markdown I do not see any code to handle this situation.

**Comment by AdUser** [comment]
> Going through the code of Text::Markdown I do not see any code to handle this situation

@jahagirdar, right, because this sample is a multi-markdown extension. So Text::**Multi**Markdown should handle it itself.

**Comment by jahagirdar** [comment]
Sorry, typo from my end, I meant MultiMarkdown

**Comment by theory** [comment]
Just ran into this myself:

    > cat content/about/test.md             
    ``` perl
    my $foo = 1;
    ```

    > MultiMarkdown.pl content/about/test.md
    <p><code>perl
    my $foo = 1;
    </code></p>

    > multimarkdown content/about/test.md  
    <pre><code class="perl">my $foo = 1;
    </code></pre>

**Comment by theory** [comment]
A more detailed example:

    > cat content/about/test.md          
    ```
    my $foo = 1;
    # This is the end.

    for my $foo (@array) {
        say $foo;
    }
    ```
    > MultiMarkdown.pl content/about/test.md
    <p>```
    my $foo = 1;</p>

    <h1 id="thisistheend.">This is the end.</h1>

    <p>for my $foo (@array) {
        say $foo;
    }
    ```</p>
    > multimarkdown content/about/test.md   
    <pre><code>my $foo = 1;
    # This is the end.

    for my $foo (@array) {
        say $foo;
    }
    </code></pre>

This is with Text::MultiMarkdown 1.000035.
